sync/config.go

```diff
@@ -8,6 +8,9 @@ import (
 
 var LatestBlockInterval = uint32(720) // 720 blocks is about two hours
 
+// IMPORTANT NOTE:
```

Comment:
This comment here is not useful. Code is very clear. 
We should inform end user. Please remove this comment and update example: https://github.com/zarbchain/zarb-go/blob/main/node/config/example_config.toml

Example config will be created if user create a new node.